sagemaker-spark-sdk/src/main/scala/com/amazonaws/services/sagemaker/sparksdk/algorithms/SageMakerAlgorithmParams.scala

```diff
@@ -55,6 +55,23 @@ private[algorithms] trait SageMakerAlgorithmParams extends Params {
       }
   }
 
+  protected def inArrayOrAboveParamValidator(validValues: Array[String], lowerBound: Double,
```

Comment:
Is the 'inclusive' parameter needed? Seems like this is only invoked once with False.